frontend/styles/globals.css

```diff
@@ -38,3 +38,19 @@ a:visited {
   color: var(--linkBlue);
   text-decoration: underline;
 }
+
+table {
+  border-collapse: collapse;
+}
+
+td,
+thead,
+tfoot {
+  font-family: Inter;
+  height: 36px;
```

Comment:
can we add `--size36: 2.25rem` to `variables.css` and use it here!

Also, I would've thought `globals.css:19` would already cover the font-family - is that not the case?

Reply:
That actually led to some confusion for me: Are we using literally the same font-family for every element on the site?

Reply:
Currently yes, it's what Ed had specified in our typography - if we don't like that we can certainly revisit it in a design convo - but that's what's in wireframes ATM